app/src/androidTest/java/ch/epfl/sweng/runpharaa/MainActivityTest.java

```diff
@@ -106,6 +110,14 @@ public void testOpenFilters() {
                 .check(matches(isDisplayed()));
     }
 
+    @Test
+    public void testOpenAndCloseHelp() {
```

Comment:
We should ensure that all tests have at least an assert

Reply:
Well the test open the popup window and close it by clicking on it, I tried to copy how the testOpenFilters() works. Isn't this enough as it passes over all the code needed to display the popup?

Reply:
Maybe check that you arrive back to your initial activity after closing the popup

Reply:
This is UI testing and there is a check to see if the popup is displayed, so the test seems good enough for me. Plus, there is no activity change so no Intent triggers to check. You could potentially verify that the popup isn't displayed anymore.

Reply:
I made it checks that the tab layout from the main activity is visible, I hope that Travis will like it. Thanks for the advice!